processors/HNL.py

```diff
@@ -165,12 +180,23 @@
 )
 
 
+
+analyzerChain.append(
+    JetMetUncertainties(
+            era = globalOptions["year"],
+            globalTag=jecTags[globalOptions["year"]],
+	    jerTag = jerTags[globalOptions["year"]],
```

Comment:
Also indents look inconsistent, could it be you are mixing whitespace and tabs?

Reply:
That is weird. it looks fine on my github...